com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceAmbientOcclusion.cs

```diff
@@ -376,42 +372,25 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                     // If true, SSAO pass is inserted after opaque pass and is expected to modulate lighting result now.
                     if (m_CurrentSettings.AfterOpaque)
                     {
-                        CoreUtils.SetRenderTarget(
-                            cmd,
-                            m_Renderer.cameraColorTargetHandle,
-                            RenderBufferLoadAction.Load,
-                            RenderBufferStoreAction.Store,
-                            m_Renderer.cameraDepthTargetHandle,
-                            RenderBufferLoadAction.Load,
-                            RenderBufferStoreAction.Store,
-                            ClearFlag.None,
-                            Color.clear
-                        );
-                        cmd.DrawMesh(RenderingUtils.fullscreenMesh, Matrix4x4.identity, m_Material, 0, (int)ShaderPasses.AfterOpaque);
+                        bool isRenderToBackBufferTarget = !cameraData.isSceneViewCamera; //&& !cameraData.isDefaultViewport;
+                        #if ENABLE_VR && ENABLE_XR_MODULE
+                        if (cameraData.xr.enabled)
+                            isRenderToBackBufferTarget = true;
+                        #endif
+                        RenderingUtils.FinalBlit(  // TODO: Only final blit if there is no FinalBlitPass called after (m_HasFinalPass in PostProcessPass)
```

Comment:
There's a TODO to solve here (@thomas-zeng)

Reply:
I am not sure I fully understand the comment here. What's the todo work here?

Reply:
IIRC The final blit is called even if there's a final pass, which is wrong.